test/integration/running/test_running.py

```diff
@@ -411,3 +411,56 @@ def test_running_reread_config(tt_cmd, tmpdir):
     # Check that the process was terminated correctly.
     instance_process_rc = instance_process.wait(1)
     assert instance_process_rc == 0
+
+
+def test_no_args_usage(tt_cmd):
+    test_app_path_src = os.path.join(os.path.dirname(__file__), "multi_app")
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        test_app_path = os.path.join(tmpdir, "multi_app")
+        shutil.copytree(test_app_path_src, test_app_path)
+
+        for subdir in ["", "multi_app"]:
+            if subdir != "":
+                os.mkdir(os.path.join(test_app_path, "multi_app"))
+            # Start all instances.
+            start_cmd = [tt_cmd, "start"]
+            instance_process = subprocess.Popen(
+                start_cmd,
+                cwd=test_app_path,
+                stderr=subprocess.STDOUT,
+                stdout=subprocess.PIPE,
+                text=True
+            )
+            for i in range(0, 3):
+                start_output = instance_process.stdout.readline()
+                assert re.search(r"Starting an instance \[app1:(router|master|replica)\]",
+                                 start_output)
+
+            start_output = instance_process.stdout.readline()
+            assert re.search(r"Starting an instance \[app2\]", start_output)
+
+            # Check status.
+            for instName in ["master", "replica", "router"]:
+                file = wait_file(os.path.join(test_app_path, run_path, "app1", instName),
+                                 instName + ".pid", [])
+                assert file != ""
+
+            file = wait_file(os.path.join(test_app_path, run_path, "app2"),
+                             "app2" + ".pid", [])
+            assert file != ""
+
+            status_cmd = [tt_cmd, "status"]
+            status_rc, status_out = run_command_and_get_output(status_cmd, cwd=test_app_path)
+            assert status_rc == 0
+            assert re.search(r"RUNNING. PID: \d+.", status_out)
```

Comment:
Status messages must have the same "app:inst" format as start messages. Check this in test.

Reply:
Fixed, tests too.